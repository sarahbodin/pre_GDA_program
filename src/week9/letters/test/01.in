5
bubble
flamingo
balloon
abba
y
