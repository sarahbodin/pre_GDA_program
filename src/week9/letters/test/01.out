Case #1: b
Case #2: f
Case #3: l
Case #4: a
Case #5: y
